fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/NumericAggregationValue.java

```diff
@@ -207,6 +208,75 @@ private static Map<Pair<LogicalOperator, TypeCode>, PhysicalOperator> computeOpe
         return mapBuilder.build();
     }
 
+    /**
+     * BitMap aggregation {@code Value}.
+     */
+    public static class BitMap extends NumericAggregationValue implements StreamableAggregateValue, IndexableAggregateValue {
```

Comment:
Naming convention `...Value`

Reply:
I see other classes are named as `Sum`, `Avg`... maybe `BitMap` is better here?

Reply:
You are right. However, please don't uncase the `m` in `bitmap`.

Reply:
changed to `BitmapConstructAgg`